src/components/CreateRules/UploadRules.vue

```diff
@@ -1,59 +1,255 @@
+      <header>
+        <h2>2. Confirm Rules Completion</h2>
+      </header>
+      <v-card flat id="confirm-rules-card">
+        <v-checkbox
+          class="chk-rules"
+          id="chk-confirm-rules"
+          v-model="rulesConfirmed"
+          label="I confirm the following items are included as required in the Rules of the Association:"
+          @change="onRulesConfirmedChange()"
+        />
+        <ul>
+          <li>The Cooperative name is identified <b>exactly</b> as follows throughout the Memorandum:</li>
+          <p><b>{{getNameRequestDetails.approvedName}}</b></p>
+          <li>Each Subscriber and Witness has signed and dated the Rules of the Association and their name is
+            printed under their signature.</li>
+        </ul>
+      </v-card>
+    </section>
+
+    <section id="upload-rules-section" class="mt-10">
+      <header>
+        <h2>3. Upload Rules</h2>
+        <ul>
+          <li>Must be set to fit onto 8 <sup>1</sup>&frasl;<sub>2</sub>" x 11" letter-size paper</li>
+          <li>Use a white background and a legible font with contrasting font colour</li>
+          <li>PDF file type (maximum 10 MB file size)</li>
+        </ul>
+        <div id="upload-rules-note">
+          <b>Note: </b>Do not upload Housing Cooperative occupancy agreements.
+        </div>
+      </header>
+      <v-card flat id="upload-rules-card">
+        <v-row>
+          <v-col id="upload-rules-card-left-col" cols="2">
+            <v-card-title class="upload-rules-vcard-title">Upload Rules</v-card-title>
+          </v-col>
+          <v-col id="upload-rules-card-right-col" cols="10">
+            <FileUploadPreview
+              :maxSize="MAX_FILE_SIZE"
+              :inputFile="uploadRulesDoc"
+              @fileSelected="fileSelected"
+              @isFileValid="isFileUploadValidFn"
+            ></FileUploadPreview>
+          </v-col>
+        </v-row>
+      </v-card>
+    </section>
+  </div>
 </template>
 
 <script lang="ts">
 // Libraries
-import { Component, Prop, Vue } from 'vue-property-decorator'
-import { Getter } from 'vuex-class'
+import { Component, Mixins, Prop, Vue } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+// Interfaces
+import {
+  ActionBindingIF,
+  CreateRulesIF,
+  CreateRulesResourceIF,
+  RulesDocIF,
+  DocumentUpload,
+  NameRequestDetailsIF
+} from '@/interfaces'
 
 // Enums
 import { RouteNames } from '@/enums'
-import { CreateRulesIF } from '@/interfaces'
 
-@Component({})
-export default class UploadRules extends Vue {
+// Mixins
+import { DocumentMixin } from '@/mixins'
+
+// Components
+import FileUploadPreview from '@/components/common/FileUploadPreview.vue'
+
+@Component({
+  components: {
+    FileUploadPreview
+  }
+})
+export default class UploadRules extends Mixins(DocumentMixin) {
   @Prop({ default: false })
   private readonly isSummary: boolean
 
+  private MAX_FILE_SIZE = 10 * 1024 // 10 MB in KB
```

Comment:
Unused prop (isSummary).

Reply:
removed `private readonly isSummary: boolean` line

Reply:
Oops, you have to delete the decorator too.

Reply:
I think you are a machine 😄

Reply:
removed decorator too

Reply:
Nah, I've made a lot of these kinds of mistakes before 😆